includes/class-wc-payments-account.php

```diff
@@ -551,7 +538,9 @@ private function get_cached_account_data() {
 				$account = [];
 				set_transient( self::ON_BOARDING_DISABLED_TRANSIENT, true, 2 * HOUR_IN_SECONDS );
 			} else {
-				throw $e;
+				// Failed to retrieve account data. Exception is logged in http client.
```

Comment:
Is it OK not to log exception here?

Reply:
Yes, it is. As the comment says the exception is already logged in other place :)

Reply:
OMG. I miss that comment big time.